openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -79,9 +79,9 @@ <h3 class="section-title">${ section['display_name'] }</h3>
                                             aria-controls="${ subsection['id'] }_contents">
                                         <span class="fa fa-chevron-right ${ 'fa-rotate-90' if subsection_is_auto_opened else '' }"
                                               aria-hidden="true"></span>
-                                        <span class="subsection-title">
+                                        <h4 class="subsection-title">
```

Comment:
Please use h3 here (highest semantic order in the page). I know it's inconsistent visually with the h3 on the aside; I think that's a separate bug.